app/components/documents/document-card.hbs

```diff
@@ -16,6 +16,23 @@
             </div>
           </div>
           <div class="vlc-document-card-toolbar__right">
+            {{#unless this.loadPieceRelatedData.isRunning}}
+              {{#if this.piece.publicationFlow}}
+                {{!-- TODO: remove style attributes and replace classes with auk-status-pill --}}
+                <LinkTo class="auk-pill auk-pill--default auk-u-mr-2"
+                  @route="publications.publication"
+                  @model={{this.piece.publicationFlow.id}}>
+                  <div class="auk-icon auk-icon--default auk-icon--link"
+                    style="display: none;">
+                    <i class="ki ki-link"></i>
+                  </div>
+                  <span class="auk-status-pill__label"
+                    style="padding: 0;">
+                    {{this.piece.publicationFlow.identification.idName}}
+                  </span>
+                </LinkTo>
+              {{/if}}
+            {{/unless}}
```

Comment:
idem hier, feature specifieke dingen gaan steken in generieke componenten willen we liever niet.
Vroeger zat die document-card (oude versie) vol met specifieke dingen voor agendaitems enzo, en na refactor door @MikiDi is deze document-card zo generiek gemaakt om te kunnen werken met eender welk model dat een piece heeft